packages/medusa/src/subscribers/search-indexing.ts

```diff
@@ -74,16 +104,7 @@ class SearchIndexingSubscriber {
           "created_at",
           "updated_at",
         ],
-        relations: [
```

Comment:
question: I think that if we do not specify the select, it will default select all the columns, wdyt?